github-dark.css

```diff
@@ -1956,7 +1956,7 @@
   .state-indicator.renamed, .discussion-topic .branch-status.status-pending,
   .compare-pr-placeholder, .compare-cutoff, .diff-cutoff, .flash.flash-warn,
   .flash-global.flash-warn, .markdown-body li.added.moved, .repo-private-label,
-  .gist-secret-label, .label-private, .stale-files-tab {
+  .gist-secret-label, .label-private, .RecentBranches, .stale-files-tab {
```

Comment:
Also, our convention has been to add any new class name to the end of the selector list.